geoviews/util.py

```diff
@@ -580,28 +579,37 @@ def process_crs(crs):
     """
     try:
         import cartopy.crs as ccrs
-        import geoviews as gv # noqa
+        import pyproj
     except ImportError:
-        raise ImportError('Geographic projection support requires GeoViews and cartopy.')
+        raise ImportError('Geographic projection support requires geoviews, pyproj and cartopy.')
 
     if crs is None:
         return ccrs.PlateCarree()
 
-    if isinstance(crs, str) and crs.lower().startswith('epsg'):
+    errors = []
+    if isinstance(crs, str):
+        starts = ['epsg:', '+init=epsg:']
+        for start in starts:
+            if crs.lower().startswith(start):
+                try:
+                    return ccrs.epsg(crs[len(start):].strip())
+                except Exception as e:
+                    errors.append(e)
+    if isinstance(crs, int):
         try:
-            crs = ccrs.epsg(crs[5:].lstrip().rstrip())
-        except Exception:
-            raise ValueError("Could not parse EPSG code as CRS, must be of the format 'EPSG: {code}.'")
-    elif isinstance(crs, int):
-        crs = ccrs.epsg(crs)
-    elif isinstance(crs, str) or is_pyproj(crs):
+            return ccrs.epsg(crs)
+        except Exception as e:
+            crs = str(crs)
```

Comment:
```suggestion
            crs = str(crs)  # this will be converted to try again below
```

Reply:
In general, I don't want to explain what the code does but why it does it. I feel like this suggestion falls into the latter category.